From 023141391debf7e09c8be240c999356b067fbaa5 Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Fri, 31 Jul 2020 15:47:50 +0800
Subject: [PATCH] ARM: dts: rv1109: correct reset pins definitions on 38 board

sensor driver use reset instead of rst, so sync this.

Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
Change-Id: I4d5a1bbf7db9da53a5ca6c5740ef1e1dbb8f796d
---
 arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts b/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
index fe15fa968494..60d336fcda39 100644
--- a/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
+++ b/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
@@ -150,7 +150,7 @@
 		dovdd-supply = <&vcc_1v8>;
 		dvdd-supply = <&vcc_dvdd>;
 		pwdn-gpios = <&gpio1 RK_PD4 GPIO_ACTIVE_HIGH>;
-		rst-gpios = <&gpio1 RK_PD5 GPIO_ACTIVE_LOW>;
+		reset-gpios = <&gpio1 RK_PD5 GPIO_ACTIVE_LOW>;
 		rockchip,camera-module-index = <1>;
 		rockchip,camera-module-facing = "front";
 		rockchip,camera-module-name = "CMK-OT1607-FV1";
-- 
2.35.3

